packages/server/server.js

```diff
@@ -3,6 +3,7 @@ const session = require("express-session");
 const MongoStore = require("connect-mongo");
 const apiRoutes = require("./routes/apiRoutes");
 const path = require("path");
+require("dotenv").config();
```

Comment:
Nice catch, weird that it wasn't here.

You might want to move this to the top